smithy-aws-apigateway-openapi/src/main/java/software/amazon/smithy/aws/apigateway/openapi/ApiGatewayConfig.java

```diff
@@ -55,7 +55,7 @@
 
     private ApiType apiGatewayType = ApiType.REST;
     private boolean disableCloudFormationSubstitution;
-    private List<String> additionalAllowedCorsHeaders = ListUtils.of("amz-sdk-invocation-id", "amz-sdk-request");
```

Comment:
Would these be deduplicated by a `set` or do they _need_ to be removed?

Reply:
That is the wrong place to have them, which is a good enough reason for me. They wouldn't be de-duped because they're lower case and the others are canonicalized.